nixos-modules/microvm/store-disk.nix

```diff
@@ -65,13 +65,15 @@ in
           pkgs.buildPackages.time
           pkgs.buildPackages.bubblewrap
           {
-            squashfs = [ pkgs.buildPackages.squashfs-tools-ng ];
```

Comment:
This only worked before because flattening nested lists like that produces an empty string. With __structuredAttrs it no longer does that.